junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ParallelExecutionConfiguration.java

```diff
@@ -60,4 +61,17 @@
 	 */
 	int getKeepAliveSeconds();
 
+	/**
+	 * Get the predicate called when the {@code ForkJoinPool} is saturated.
+	 * This occurs when the ForkJoinPool has reached MaxPoolSize
+	 * and there are no worker threads available. The predicate
+	 * should return true if the calling thread should block.
+	 * If the predicate is null or returns false a RejectedExcecutionException
+	 * is thrown.
+	 * @return a Predicate accepting a ForkJoinPool as the parameter or null.
+	 * @since 1.9
+	 * @API(since = "1.9")
+	 */
+	Predicate<? super ForkJoinPool> getSaturatePredicate();
```

Comment:
This needs to be a `default` method that returns `null` to preserve backwards compatibility.
```suggestion
	default Predicate<? super ForkJoinPool> getSaturatePredicate() {
		return null;
	}
```

Reply:
Done